types/jsdom/index.d.ts

```diff
@@ -1,6 +1,7 @@
 // Type definitions for jsdom 11.0
```

Comment:
`pretendToBeVisual` was [added in 11.4.0](https://github.com/jsdom/jsdom/commit/8a6894c34f14b9131d0fe4acadb71dae4f49daca) and `storageQuota` was [added in 11.12.0](https://github.com/jsdom/jsdom/commit/23d67ebec901b3471b84e63f58a96b51a36f3671). Please change the version number to 11.12 to account for this.

Reply:
Good points, thanks! Will do